pages/common/hg-serve.md

```diff
@@ -0,0 +1,27 @@
+# hg serve
+
+> Start a standalone Mercurial web server for browsing repositories.
+
+- Start a web server instance:
+
+`hg serve`
+
+- Start a web server instance on the specified port:
+
+`hg serve --port {{port}}`
+
+- Start a web server instance on the specified listening address:
+
+`hg serve --address {{address}}`
+
+- Start a web server instance with a specific name:
+
+`hg serve --name {{name}}`
+
+- Start a web server instance using the specified theme:
```

Comment:
Is there a predefined list of themes one can use ? Where do I get the list of themes ?

Reply:
It is predefined, but you can also specify a custom theme if it's added to the template directory.

The downside is that there is no way (as far as I know) to list themes, other than to do an `ls` in the themes directory.

Reply:
Ok, then perhaps we can mention that in brackets.